Google.PowerShell/ReleaseFiles/GoogleCloudPlatform.Format.ps1xml

```diff
@@ -162,5 +162,52 @@
         </TableRowEntries>
       </TableControl>
     </View>
+    <View>
+      <Name>BigQuery Data Row</Name>
```

Comment:
That's totally doable, but then what it ends up looking like is 

Title, Author, Year
Omelette, Shakespeare, 1603
Title, Author, Year
Omelette, Shakespeare, 1603
Title, Author, Year
Omelette, Shakespeare, 1603

...etc.  I tried to control the formatting of the List<BigQueryRow>, but that didn't work well at all.  Jim sent me a few links, and I'm looking into what options we have.